src/containers/Admin/EditTab/ToggleDetailsPanel/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { Fieldset } from '@entur/form'
+import { FilterChip } from '@entur/chip'
+import { Label } from '@entur/typography'
+import './styles.scss'
+
+import { useSettingsContext } from '../../../../settings'
+
+function ToggleDetailsPanel(): JSX.Element {
+    const [settings, settingsSetters] = useSettingsContext()
+    const { hideSituations } = settings || {}
+    const { setHideSituations } = settingsSetters
+    return (
+        <Fieldset className="toggleDetail-panel">
+            <div>
+                <FilterChip
+                    className="filterChipComp"
```

Comment:
Klassenavn bør bruke kebab-case, ikkje camelCase-format. Sjekk gjerne getbem.com for meir info om BEM-konvensjonen for navngivning av klassar.